src/style-spec/reference/v8.json

```diff
@@ -4392,6 +4392,27 @@
         "data-driven styling": {}
       }
     },
+    "raster-scaling": {
```

Comment:
What do you think about renaming this property to `raster-filter`, `raster-mag-filter`, or something along those lines so the property name is in line with the existing WebGL terminology?

Reply:
| tool | name |
|---|---|
| carto | _raster-scaling_ [ref](http://cartocss.readthedocs.io/en/latest/mapnik_api.html?highlight=raster#raster-scaling) |
| GIMP | _interpolation_ (none (nearest) and linear)
| ImageMagick | _resampling filter_ |
| OpenGL | _texture filtering_ |
| gdal | _resampling method_ (near, bilinear) |
| qgis | _resampling_ |
| rasterio | _resampling method_ [ref](https://rasterio.readthedocs.io/en/latest/topics/resampling.html#resampling-methods) |
| css | _image-rendering_ [ref](https://developer.mozilla.org/en-US/docs/Web/CSS/image-rendering)

I'm fine with renaming. `raster-filter` or `raster-resampling` both seem to be the most widely used.

I don't think terminology necessarily needs to match the underlying implementation's.

I'm leaning more towards `raster-resampling` due to it's popularity with raster GIS tools, but I'm not fussed either way...

Reply:
@kkaefer not sure if you saw this thread as well? Resampling and filtering are quite popular too...